CHANGELOG.md

```diff
@@ -1,25 +1,43 @@
-# Unreleased
+# Unreleased: 12.0.0 (Breaking release)
 
-## Features
+## Breaking changes 
 
-### Preserve query strings
+This release ensures the GOV.UK Prototype Kit reflects the latest release of the GOV.UK Frontend, v4.0.0.
 
-We now preserve URL query strings when redirecting POST requests to GET requests.
+### Update to GOV.UK Frontend v4.0.0 [(#1195)](https://github.com/alphagov/govuk-prototype-kit/pull/1195)
```

Comment:
```suggestion
### Update to GOV.UK Frontend v4.0.0
```
Reason for suggestion: what we've done before in Frontend/Kit notes is, link to the PR from the last line in the section, usually beginning "This was added in..."

Reply:
Re relocating the link, have made suggestion on line 15. :)